brjs-sdk/sdk/libs/javascript/br-test/src/br/test/GwtTestRunner.js

```diff
@@ -243,6 +243,15 @@ GwtTestRunner.prototype.endTest = function() {
 
 	}
 
+	if (this.m_oFixtureFactory.tearDown) {
+		try {
+			this.m_oFixtureFactory.tearDown();
+		} catch (e) {
+			throw new Errors.CustomError("TestTearDownError", e.message,
```

Comment:
I don't know, I'm not sure I get the point of catching an error and then throwing your own. Why not just let the original error bubble up?